web/cypress/e2e/onboarding-as-starting-business.spec.ts

```diff
@@ -0,0 +1,84 @@
+import { randomElementFromArray } from "@businessnjgovnavigator/cypress/support/helpers/helpers";
+import { onOnboardingPageStartingBusiness } from "@businessnjgovnavigator/cypress/support/page_objects/onboardingPageWhenStarting";
+import { Industries, Industry, carServiceOptions, randomInt } from "@businessnjgovnavigator/shared/";
+
+const enabledIndustries = Industries.filter((element: Industry) => {
+  return element.isEnabled;
+});
+
+describe("Onboarding for all industries when starting a business [feature] [all] [group4]", () => {
```

Comment:
Did y'all pick group4 for any particular reason? Noticing that as a part of this work there are tests removed from groups 1, 2, and 4.